src/vs/workbench/contrib/chat/browser/chatEditing/chatEditingService.ts

```diff
@@ -136,6 +136,7 @@ export class ChatEditingService extends Disposable implements IChatEditingServic
 			return this._currentSessionObs.read(r)?.canRedo.read(r) || false;
 		}));
 		this._register(this._chatService.onDidDisposeSession((e) => {
+			this._applyingChatEditsFailedContextKey.set(false);
```

Comment:
This could have been fired for some other chat session, not your session, right?

Reply:
Yeah

Reply:
And that's a problem right? Or no?